tests/framework/Optimizers/SimulatedAnnealing/BoltzmanSA/test_simulatedAnnealing.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" ?>
 <Simulation verbosity="debug">
   <TestInfo>
-    <name>framework/Optimizers.SimulatedAnnealing.Boltzmann</name>
+    <name>framework/Optimizers.SimulatedAnnealing.BoltzmannSA</name>
```

Comment:
it should be

framework/Optimizers/SimulatedAnnealing.BoltzmannSA

Reply:
Even if the test is inside BoltzmanSA and is called boltzmanSA in the file tests?